ideaforce/src/main/java/com/emc/ideaforce/controller/CommonController.java

```diff
@@ -135,8 +139,13 @@ else if (principal != null) {
     }
 
     @GetMapping("/")
-    public String index(Principal principal) {
-        return HOME_VIEW;
+    public ModelAndView index(Principal principal) {
+        return new ModelAndView("index");
```

Comment:
Why is this changed? Where there any issue?

Reply:
I was passing isUserAdmin attribute into index. While refactoring i left it as ModelView..